src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -143,7 +145,12 @@ class DialogmoteService(
                     varselUuid = motedeltakerArbeidstakerVarselIdPair.second,
                 )
 
-                // TODO: Implement DialogmoteInnkalling-Varsel to Arbeidsgiver
+                naermesteLederVarselService.sendVarsel(
+                    createdAt = LocalDateTime.now(),
+                    naermesteLeder = narmesteLeder,
+                    varseltype = MotedeltakerVarselType.INNKALT,
+                    varselUuid = motedeltakerArbeidstakerVarselIdPair.second,
```

Comment:
`motedeltakerArbeidstakerVarselIdPair ` er en er en referanse til et varsel som er sendt til Arbeidstaker. Her sendes det derimot et varsel til en NL(motedeltakerType er da Arbeidsgiver og ikke Arbeidstaker).

Reply:
PS: denne uuid'en tror jeg ikke er relevant for `naermesteLederVarselService`

Reply:
Ps: Om vi skal sende med en UUID til narmestelederService som er referanse til varselet til Arbeidsgiver, så trenger vi å opprettet en flyt for MOTEDELTAKER_ARBEIDSGIVER_VARSEL tilsvarende som det som er gjort for MOTEDELTAKER_ARBEIDSTAKER_VARSEL.
Kan jo fjerne uuid-referansen for denne oppgaven sin skyld og så kan de legges til med resterende funksjonalitet.